README.md

```diff
@@ -1,2 +1,73 @@
-# treeherder-tests
-Website automation for Treeherder https://github.com/mozilla/treeherder-service
+Treeherder Tests
+============================
+Website automation for Treeherder for [https://treeherder.mozilla.org/][MOZ]
+
+[MOZ]: https://treeherder.mozilla.org/
+
+Running Tests
+-------------
+
+### Python
+Before you will be able to run these tests you will need to have Python 2.6 installed.
+
+If you are running on Ubuntu/Debian you will need to first do
+
+    sudo apt-get install python-setuptools
+
+to install the required Python libraries.
+
+__note__
+
+The instructions below will install the required Python libraries into your
+global Python installation. If you work on multiple Python projects that might
+end up needing different versions of the same libraries, you might want to
+follow `sudo easy_install pip` with `sudo pip install virtualenv`, and then
+create and activate a [virtualenv](http://www.virtualenv.org) (e.g. `virtualenv
+mcom-tests-env; source mcom-tests-env/bin/activate`) to create a clean
+"virtual environment" for just this project. Then you can
+`pip install -r requirements.txt` in your virtual environment
+without needing to use `sudo`.
+
+If you don't mind installing globally, just run
+
+    sudo easy_install pip
+
+followed by
+
+    sudo pip install -r requirements.txt
+
+__note__
+
+
+
+### Running tests locally
+
+To run tests locally, it's a simple case of calling the command below from this directory
+
+    py.test --driver=firefox
```

Comment:
I wonder if we should add `--baseurl=https://https://treeherder.allizom.org` here? I get an error baseurl is missing when I run otherwise.